src/node_os.cc

```diff
@@ -23,19 +23,20 @@
 #include "string_bytes.h"
 #include "util.h"
 
-#include <array>
-#include <cerrno>
-#include <cstring>
+#include <errno.h>
```

Comment:
change back to original ?
```cpp 
#include <cerrno>
#include <cstring>
```
see https://github.com/nodejs/node/pull/26159.

Reply:
Fixed.
(probably was a wrong conflict resolution, made when rebasing over #26159)